docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mambaorg/micromamba:1.5.10
+FROM mambaorg/micromamba:2.0.2
 
 ENV PGDATA=${CONTAINER_HOME}/pgdata
```

Comment:
I guess we've just been putting the Postgres data in `/pgdata` this whole time? That seems fine, I guess, though we should probably move this down to where `CONTAINER_HOME` *is* defined.

Reply:
I suspect that we need `$PGDATA` to be defined prior to installing postgres and creating a postgres cluster, so maybe it would be better to move the definition of `$CONTAINER_HOME` up to the top.

Reply:
Yes, that's probably better!